cookbook/doctrine/registration_form.rst

```diff
@@ -138,230 +170,143 @@ Next, create the form for the ``User`` model::
 
-    .. code-block:: yaml
+            <!-- app/config/routing.xml -->
+            <?xml version="1.0" encoding="UTF-8" ?>
+            <routes xmlns="http://symfony.com/schema/routing"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
 
-        # src/Acme/AccountBundle/Resources/config/routing.yml
-        account_register:
-            path:     /register
-            defaults: { _controller: AcmeAccountBundle:Account:register }
+                <route id="user_registration" path="/register">
+                    <default key="_controller">AppBundle:Registration:register</default>
+                </route>
+            </routes>
 
-        account_create:
-            path:     /register/create
-            defaults: { _controller: AcmeAccountBundle:Account:create }
+        .. code-block:: php
 
-    .. code-block:: xml
+            // app/config/routing.php
+            use Symfony\Component\Routing\RouteCollection;
+            use Symfony\Component\Routing\Route;
 
-        <!-- src/Acme/AccountBundle/Resources/config/routing.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <routes xmlns="http://symfony.com/schema/routing"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+            $collection = new RouteCollection();
+            $collection->add('user_registration', new Route('/register', array(
+                '_controller' => 'AppBundle:Registration:register',
+            )));
 
-            <route id="account_register" path="/register">
-                <default key="_controller">AcmeAccountBundle:Account:register</default>
-            </route>
+            return $collection;
 
-            <route id="account_create" path="/register/create">
-                <default key="_controller">AcmeAccountBundle:Account:create</default>
-            </route>
-        </routes>
+Next, create the template:
 
-    .. code-block:: php
+.. code-block:: html+jinja
 
-        // src/Acme/AccountBundle/Resources/config/routing.php
-        use Symfony\Component\Routing\RouteCollection;
-        use Symfony\Component\Routing\Route;
+    {# app/Resources/views/registration/register.html.twig #}
+    
+    {{ form_start(form) }}
+        {{ form_row('form.username') }}
+        {{ form_row('form.email') }}
 
-        $collection = new RouteCollection();
-        $collection->add('account_register', new Route('/register', array(
-            '_controller' => 'AcmeAccountBundle:Account:register',
-        )));
-        $collection->add('account_create', new Route('/register/create', array(
-            '_controller' => 'AcmeAccountBundle:Account:create',
-        )));
+        {{ form_row('form.plainPassword.first', {
+            'label': 'Password'
```

Comment:
Just a suggestion: could we set the labels in the form class and therefore simplify this template drastically? We could then add a link to the Form Twig customization article that explains `form_row()`, etc.